frontend/src/components/camp/CollaboratorListItem.vue

```diff
@@ -5,26 +5,36 @@
     </v-list-item-avatar>
     <v-list-item-content>
       <v-list-item-title>{{ collaborator.user().username }}</v-list-item-title>
-      <v-list-item-subtitle>{{ collaborator.user().mail }}</v-list-item-subtitle>
+      <v-list-item-subtitle>
+        {{
+          collaborator.user().mail
+        }}
+      </v-list-item-subtitle>
     </v-list-item-content>
     <api-select
       :value="collaborator.role"
       :uri="collaborator._meta.self"
       fieldname="role"
-      :items="['member', 'manager']"
+      :items="[
+        { key: 'member', translation: $tc('entity.camp.collaborators.member') },
+        { key: 'manager', translation: $tc('entity.camp.collaborators.manager') },
+      ]"
+      item-value="key"
+      item-text="translation"
       :my="0"
       dense
       vee-rules="required" />
     <v-list-item-action class="ml-2">
       <button-delete @click="api.del(collaborator)">
-        Remove
+        {{ $tc("components.camp.collaborators.remove") }}
```

Comment:
Translation key should be `components.camp.collaboratorListItem.remove` (also remember to change it in the JSON)